src/main/resources/avro/bdg.avdl

```diff
@@ -472,44 +478,99 @@ enum GenotypeType {
 // sample but are computed based on the samples.  For instance,  MAPQ0 is an aggregate
 // stat computed from all samples and stored inside the INFO line.
 record VariantCallingAnnotations {
-  // QUAL: Phred-scaled probability of error for this variant call.
-  union { null, float }   variantCallErrorProbability = null;
 
   // FILTER: True or false implies that filters were applied and this variant PASSed or not.
   // While 'null' implies not filters were applied.
   union { null, boolean } variantIsPassing = null;
   array <string> variantFilters = [];
 
-  union { null, int }     readDepth = null;
+  /**
+   True if the reads covering this site were randomly downsampled to reduce coverage.
+   */
   union { null, boolean } downsampled = null;
-  union { null, float }   baseQRankSum = null;
-  union { null, float }   clippingRankSum = null;
-  union { null, float }   fisherStrandBiasPValue = null; // Phred-scaled.
-  union { null, float }   haplotypeScore = null;
-  union { null, float }   inbreedingCoefficient = null;
-  union { null, float }   rmsMapQ = null;
-  union { null, int }     mapq0Reads = null;
-  union { null, float }   mqRankSum = null;
-  union { null, float }   variantQualityByDepth = null;
-  union { null, float }   readPositionRankSum = null;
+
+  /**
+   The Wilcoxon rank-sum test statistic of the base quality scores. The base quality
+   scores are separated by whether or not the base supports the reference or the
+   alternate allele.
+   */
+  union { null, float } baseQRankSum = null;
+
+  /**
+   */
+  union { null, float } clippingRankSum = null;
```

Comment:
I don't think this field is used anywhere (and I don't have good docs) so I'm going to drop it.